source/beamline/CLS/CLSValves.h

```diff
@@ -16,6 +23,9 @@ class CLSValves : public CLSTriStateGroup
 	/// Destructor.
 	virtual ~CLSValves();
 
+	/// helper function to create a beam on action list
+	AMListAction3* createBeamOnActionList();
+
```

Comment:
I was thinking more as a general CLSValves aspect rather than being a coupled part of beam status.  Generally, the use case for closing valves is minimal but I didn't want us to not think about it.